src/style.css

```
@@ -1 +1,72 @@
-/* Styles go here */
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
```

```suggestion
  padding: 0 50px;
```
Можна трохи скоротити